src/pivot-table/components/cells/utils/get-adjuster-color.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-unsafe-enum-comparison */
+import { COLORING } from "@qlik/nebula-table-utils/lib/utils";
+import { color } from "d3-color";
+import type { StyleService } from "../../../../types/types";
+import { Colors } from "../../shared-styles";
+
+/**
+ * Converts border color to opaque color,
+ * by alpha blending the border color with the background the header or top grid background.
+ * The background is assumed to be opaque.
+ * If the background is transparent (which is the default color) it is assumed to be white
+ */
+export default function getAdjusterColor(styleService: StyleService, isHeader: boolean) {
+  const background = isHeader ? styleService.header.background : styleService.dimensionValues.background;
```

Comment:
Note that this wont work when the `isHeader` is false and the top dimension have background set via color by expression.

I think it's fine that we cannot handle that, the question is if it's better to do nothing in that case or use `styleService.dimensionValues.background` anyway?

Reply:
I think I can fix that pretty easily, will get on it.